### PR TITLE
Generally brings about half the heretic antag code up to our code standards, more or less

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

All single letter vars I saw were killed unless it was an interation or an index

some long lists given space

deantagging and antagging logging made clear

goodbye messages moved to farewell, unnecessary silent check from it

this isn't any standard but i did add spaces to some lists that didn't have them

a TON of "as anything" was added

some areas had just way too much open spacing for what they were doing, like spacing between every early return

miscellaneous code improvements

misc spelling improvements

## Why It's Good For The Game

Code improvement!

## Changelog
:cl:
code: heretic code is better! Half of it.
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
I'm so glad you didn't steal the thunder from my heretic fix

File: code/_globalvars/game_modes.dm [hunk 1]
```diff
@@ -11,5 +11,5 @@ GLOBAL_DATUM(start_state, /datum/station_state) // Used in round-end report
 //TODO clear this one up too
 GLOBAL_DATUM(cult_narsie, /obj/narsie)
 
-///We want reality_smash_tracker to exist only once and be accesable from anywhere.
+///We want reality_smash_tracker to exist only once and be accessable from anywhere.
```
Comment (celotajstg) [inline code/_globalvars/game_modes.dm:14]: accessible
https://www.thefreedictionary.com/accessable

File: code/modules/antagonists/eldritch_cult/eldritch_antag.dm [hunk 2]
```diff
@@ -26,80 +26,82 @@
 	to_chat(owner, "<span class='cult'>The book whispers softly, its forbidden knowledge walks this plane once again!<br>\
 	Your book allows you to research abilities. Read it very carefully, for you cannot undo what has been done!<br>\
 	You gain charges by either collecting influences or sacrificing people tracked by the living heart.<br> \
-	You can find a basic guide at : https://tgstation13.org/wiki/Heresy_101 </span>")
+	You can find a basic guide at: https://tgstation13.org/wiki/Heresy_101 </span>")
```
Comment (Fikou) [inline code/modules/antagonists/eldritch_cult/eldritch_antag.dm:29]: this should be a get_policy

File: code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm [hunk 3]
```diff
@@ -29,9 +29,8 @@
 /datum/eldritch_knowledge/New()
 	. = ..()
 	var/list/temp_list
-	for(var/X in required_atoms)
-		var/atom/A = X
-		temp_list += list(typesof(A))
+	for(var/atom/required_atom in required_atoms)
```
Comment (ninjanomnom) [inline code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm:32]: Unless this is supposed to be a filtered list, keeping the same behavior would be this
```suggestion
	for(var/atom/required_atom as anything in required_atoms)
```

File: code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm [hunk 4]
```diff
@@ -78,13 +77,11 @@
  *
  * By default this proc creates atoms from result_atoms list. Override this is you want something else to happen.
  */
-/datum/eldritch_knowledge/proc/on_finished_recipe(mob/living/user,list/atoms,loc)
-	if(result_atoms.len == 0)
+/datum/eldritch_knowledge/proc/on_finished_recipe(mob/living/user, list/atoms, loc)
+	if(!result_atoms.len)
```
Comment (ninjanomnom) [inline code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm:81]:
```suggestion
	if(!length(result_atoms))
```

File: code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm [hunk 5]
```diff
@@ -160,8 +155,7 @@
 
 	var/list/compiled_list = list()
 
-	for(var/H in GLOB.human_list)
-		var/mob/living/carbon/human/human_to_check = H
+	for(var/mob/living/carbon/human/human_to_check in GLOB.human_list)
```
Comment (ninjanomnom) [inline code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm:158]:
```suggestion
	for(var/mob/living/carbon/human/human_to_check as anything in GLOB.human_list)
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm (hunk 3 above)
Comment (Mothblocks) [reply code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm:32]: @ninjanomnom Remember to draft when you make code review, please.

**Pull request merged**